.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
+branch = True
+source = pycsw
+
+[paths]
+source =
+    pycsw
+    .tox/*/lib/python*/site-packages/pycsw
+    /usr/local/lib/python*/dist-packages/pycsw
```

Comment:
I believe so. 

- The second line refers to paths inside the virtualenvs generated by tox, so they are always the same, and they do not depend on the base system.
- The third line is the standard path where python packages are installed whenever they are pip installed in a debian-like system. I'm not so sure this is the same on other distros. However, I don't think it is relevant, as this configuration's purpose is for allowing the coverage output's to be treated as the same by `coverage.py`, specifically for the continuous integration service. Since we are using Travis, which is based on ubuntu, the paths are the expected ones.